packages/manager/modules/preloader/src/style.css

```diff
@@ -0,0 +1,70 @@
+#managerPreloader {
```

Comment:
Move from `buildStyles` (initially in `packages/manager/modules/preloader/src/template.js`). Don't require to dynamically build css, so I move it to CSS file.